ninchatsdk/src/main/java/com/ninchat/sdk/ninchatreview/presenter/NinchatReviewPresenter.kt

```diff
@@ -96,8 +96,8 @@ class NinchatReviewPresenter(
     }
 
     private fun renderText(view: View, rootActivity: View) {
-        view.ninchat_review_title.text = Misc.toRichText(ninchatReviewModel.getThanksYouText(), view.ninchat_review_title)
-        view.ninchat_review_description.text = Misc.toRichText(ninchatReviewModel.getFeedbackTitleText(), view.ninchat_review_description)
+        view.ninchat_review_title.text = Misc.toRichText(ninchatReviewModel.getFeedbackTitleText(), view.ninchat_review_title)
+        // view.ninchat_review_description.text = Misc.toRichText(ninchatReviewModel.getFeedbackTitleText(), view.ninchat_review_description)
```

Comment:
This can be removed, right?

Reply:
Yes, thanks. Instead of commented out the line, I now deleted it